### PR TITLE
Test: Do not merge.

### PR DESCRIPTION
**Pull request by russjones**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -395,8 +395,8 @@ test-go: FLAGS ?= '-race'
 test-go: PACKAGES := $(shell go list ./... | grep -v integration)
 test-go: CHAOS_FOLDERS := $(shell find . -type f -name '*chaos*.go' -not -path '*/vendor/*' | xargs dirname | uniq)
 test-go: $(VERSRC)
-	$(CGOFLAG) go test -tags "$(PAM_TAG) $(FIPS_TAG) $(BPF_TAG)" $(PACKAGES) $(FLAGS) $(ADDFLAGS)
-	$(CGOFLAG) go test -tags "$(PAM_TAG) $(FIPS_TAG) $(BPF_TAG)" -test.run=TestChaos $(CHAOS_FOLDERS) -cover $(ADDFLAGS)
+	$(CGOFLAG) go test -p 1 -tags "$(PAM_TAG) $(FIPS_TAG) $(BPF_TAG)" $(PACKAGES) $(FLAGS) $(ADDFLAGS)
```
Comment (r0mant) [inline Makefile:398]: Is `-p` same as `-parallel`? Couldn't find the flag description anywhere in `go help test`.

**Review by r0mant — COMMENTED** [review]
I don't mind merging this in if it helps with tests stability. I'm curious, does this affect tests running time? I.e. does CI take longer and if so by how much?

File: Makefile (hunk 1 above)
Comment (russjones) [reply Makefile:398]: Yeah, it's `-parallel` and it does make tests slower, but they appear to pass at a higher rate, so overall time to test passing is faster.

Comment (nklaassen) [reply Makefile:398]: So it looks like `-p 1` runs everything serially, including build and testing each individual package serially, while `--parallel 1` will only run tests within each package serially, multiple packages can still be tested in parallel. The `-p` flag is not documented but that's what I'm reading here https://stackoverflow.com/a/23840896/5859521

**Pull request closed without merging**